naughty/rhel-8/3724-vdo-kernel-hang-2

```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
```

Comment:
This pattern is waay to generic. But I guess it is one of those tests that there is nothing else we can grasp on?

Reply:
Unfortunately yes. It at least restricts it to the vdo test. See the [failing log](https://cockpit-logs.us-east-1.linodeobjects.com/pull-3722-20220809-231524-7e2e1e9d-rhel-8-7-distropkg-cockpit-project-cockpit/log.html#233-2). When it happens, the machine freezes completely.

I could include the `RuntimeError: Timed out on 'journalctl|gzip'` into the pattern, which would ensure "it's dead, Jim!"?

Reply:
I don't think that would make it any more specific for this exact issue. So I am fine with this, thanks!